### PR TITLE
Add Mobile Support.

### PR DESCRIPTION
**Pull request by libremente**
Decorated the student view with the XBlocks_supports for multi_device
in order to solve the compatibility issue with mobile handheld devices.

@stvstnfrd @caesar2164

### CODE REVIEW

**Comment by stvstnfrd** [comment]
+1

File: edxadapt/edxadapt.py [hunk 1]
```diff
@@ -128,7 +128,11 @@ def get_course_id(self):
         # NOTE(idegtiarov) drop meaningless part of default course_id to decrease length of id which is stored in
         # the database
         return self.course_id.course.replace('course-v1:', '')
-
+    
+    # Decorate the view in order to support multiple devices e.g. mobile
+    # See: https://openedx.atlassian.net/wiki/display/MA/Course+Blocks+API
```
Comment (z4y4ts) [inline edxadapt/edxadapt.py]: This link doesn't work for me. It says "Page Not Found"

Any chance you have another link?

Comment (libremente) [reply edxadapt/edxadapt.py]: Hi, thanks for pointing it out. It looks like they are changing the structure of their wiki and I can't find any reference to that specific page. I'll dig a bit further. Anyway, this is the commit where they introduced the `multi_device` support: https://github.com/edx/edx-platform/commit/5a44f391b0011b35cf7b9d3f8b6bba8cfe86a7b5. 
They are referring to `MA-977` but I can't find any real reference to that on the wiki.

Comment (libremente) [reply edxadapt/edxadapt.py]: @z4y4ts here it is: https://openedx.atlassian.net/wiki/display/EDUCATOR/Course+Blocks+API#CourseBlocksAPI-BlockRendering:Web,Responsive,andNative
I will make another commit, thanks for noticing.

Comment (z4y4ts) [reply edxadapt/edxadapt.py]: @libremente Great! Thanks for the update.

Comment (libremente) [reply edxadapt/edxadapt.py]: @z4y4ts corrected in the new commit, thanks!
https://github.com/Stanford-Online/EdxAdaptXBlock/pull/2/commits/e7a3588d9dcdb01f954ed51bd787624be994e407#diff-4591d93c0563558178c3d28b2f945b8bR133

**Comment by stvstnfrd** [comment]
@caesar2164 Are we running a custom copy of this currently?

If not, should we close this?

**Comment by caesar2164** [comment]
@stvstnfrd, we are running a local Stanford version of this.

**Comment by caesar2164** [comment]
Looks like we forked to pull a specific commit in early: https://github.com/Stanford-Online/EdxAdaptXBlock/commits/master

**Comment by stvstnfrd** [comment]
We didn't build it and no one is asking for mobile support; I'm inclined to drop this..

**Pull request closed without merging**